testsuite/test-blacklist.c

```diff
@@ -83,7 +83,7 @@ static int blacklist_1(const struct test *t)
 
 DEFINE_TEST(blacklist_1, .description = "check if modules are correctly blacklisted",
 	    .config = {
-		    [TC_ROOTFS] = TESTSUITE_ROOTFS "test-blacklist/",
```

Comment:
Idea overall looks nice. I like the dedup here. However as I said I´d just do open(TESTSUITE_ROOTFS) and then use openat() to make everything relative to that.